soccer/gameplay/plays/defense/pileup.py

```diff
@@ -0,0 +1,115 @@
+import robocup
+import constants
+import play
+import skills
+import tactics
+import main
+import evaluation
+import math
+import random
+
+'''
+def fighting(r, x, y):    
+    d = sqrt(abs(x - r.pos.x) + abs(y - r.pos.y))
+    return d <= (3 * constants.Robot.Radius + constants.Ball.Radius)
+'''
+
+class Pileup(play.Play):
+
+    def __init__(self):
+        super().__init__(continuous=False)
+        
+        x = main.ball().pos.x #x-coordinate of ball's position
+        y = main.ball().pos.y #y-coordinate of ball's position
+        us = main.our_robots()
+        them = main.their_robots()
+
+        ''' THIS IS TO BE IMPLEMENTED NEXT. GOAL IS TO CALCULATE AVERAGE POSITION OF OUR BOTS AND THEIR BOTS TO DETERMINE WHERE TO PLACE COLLECTING ROBOTS IN THE EVENT OF A GIVEN PILEUP
```

Comment:
Is this something you're still planning on doing?  If it is, I think  there might already be a function for this somewhere, I would ask on the robocup slack to see if anyone knows

Reply:
For the time being, I'll keep the basic version, but I'll ask around sometime next week to try and add this to this play.